### DISCUSSION TITLE
Offer ability to show other EXIF information (City, state)

### DISCUSSION THREAD

**Discussion by jschwalbe** [damongolding/immich-kiosk#45] [Ideas]
There are many pictures with an obvious location, but there are also many where I wonder where I was when that pic was taken. Would love to have things like City & State show up on my kiosk. It looks like you've already accounted for it in the code, I'm just not sure how to bring it out to be displayed.

**Comment by damongolding** [comment]
I have display meta data (aperture, ISO etc) on the TODO list on the README but I could easily add that info into that job, as long as the info is available of course!

**Comment by damongolding** [comment]
Added in v0.8.0
